docs/src/examples/components/Button/Performance/Button.perf.tsx

```diff
@@ -3,4 +3,6 @@ import * as React from 'react'
 
 const ButtonPerf = () => <Button content="Click here" />
 
+ButtonPerf.iterations = 5000
```

Comment:
Do we need to add it explicitly everywhere or can we use 5000 as a default?

Reply:
Good question, guess it depends on the overall strategy regarding writing perf scenarios. It's definitely too high for most of the perf scenarios already written in docs. Maybe it's a good practice to default to 1 to force perf story writers to consider what they're testing rather than being one-dimensional and repetitive?

For now I've left it at 1 as I did not want arbitrary new perf stories adding significant time to build / CI.